e2e/clients/juno/client.star

```diff
@@ -15,6 +15,7 @@ def run(plan, name, participant):
         "--http-host", "0.0.0.0",
         "--log-level", "debug",
         "--db-path", "/var/lib/juno",
+        "--disable-l1-verification"
```

Comment:
The issue is that, in the current pipeline, tests are run by default on the latest released juno version ([nethermind/juno:latest](https://github.com/NethermindEth/starknet-p2p-tests/blob/87c65a0b1c4d99a328e7bd6923c3fabbce9f3dcc/e2e/clients/juno/client.star#L4)), which doesn’t support this flag yet so [pipeline](https://github.com/NethermindEth/starknet-p2p-tests/actions/runs/11673417331/job/32504096641#step:4:39) gonna fail.

Reply:
oh, we are getting this from Docker hub instead of artifactory, right?

Reply:
yep